src/System.Drawing.Primitives/tests/ColorTests.cs

```diff
@@ -491,5 +494,45 @@ public void DebuggerAttributesAreValid()
             DebuggerAttributes.ValidateDebuggerDisplayReferences(Color.Aquamarine);
             DebuggerAttributes.ValidateDebuggerDisplayReferences(Color.FromArgb(4, 3, 2, 1));
         }
+
+        [ConditionalFact(nameof(SupportsSystemEvents))]
+        public void UserPreferenceChangingEventTest()
+        {
+            int element = 12; // Win32SystemColors.AppWorkSpace.
+            Color oldColor = System.Drawing.SystemColors.AppWorkspace;
+
+            // A call to ToArgb is necessary before changing the system colors because it initializes the knownColorTable.
+            int oldColorArgb = oldColor.ToArgb();
+            int oldColorAbgr = GetColorRefValue(oldColor);
+
+            Color newColor = oldColor != Color.Gold ? Color.Gold : Color.Silver;
+            int newColorArgb = newColor.ToArgb();
+            int newColorAbgr = GetColorRefValue(newColor);
```

Comment:
nit: technically this should be newColorBgr not Agbr, since it omits the alpha channel.